h/accounts/views.py

```diff
@@ -245,7 +245,7 @@ def __init__(self, request):
         self.schema = schemas.ResetPasswordSchema().bind(request=self.request)
         self.form = request.create_form(
             schema=self.schema,
-            action=self.request.route_path('account_reset'),
+            action=self.request.route_path('account/reset'),
```

Comment:
`request.route_path` [takes the route name as the first argument](http://docs.pylonsproject.org/projects/pyramid/en/latest/api/request.html#pyramid.request.Request.route_path), so this should be `account_reset`.